internal/pkg/agent/cmd/container.go

```diff
@@ -789,10 +789,9 @@ func setPaths(statePath, configPath, logsPath string, writePaths bool) error {
 	}
 	originalInstall := paths.Install()
 	originalTop := paths.Top()
-	paths.SetTop(topPath)
+	paths.SetTop(statePath) // change this?
 	paths.SetConfig(configPath)
-	// when custom top path is provided the home directory is not versioned
-	paths.SetVersionHome(false)
```

Comment:
I'm curious to know what this does (I expect we need Michal or Blake to answer this). It's a bit hard to tell from the code.

As far as I can tell in the container I built, the home path is versioned by default. I get `/usr/share/elastic-agent/data/elastic-agent-fc3eba`. I don't know that this would change if the STATE_PATH were to change either so removing it seems right but it would be good to confirm.

Reply:
Yah, this just adds the `elastic-agent-fc3eba` directory, which is still there in docker.